### ISSUE TITLE
Can you provide the config file of "Text guided image editing by VQ-Diffusion"?

### ISSUE THREAD

**Issue by Yikai-Wang** [cientgu/VQ-Diffusion#6]
Hi there, 
I am interested in your excellent work and try to understand your code. Can you provide the config file to run inference on Figure 5 (Text guided image editing by VQ-Diffusion.)? It will be very helpful. Thanks!

**Comment by cientgu** [comment]
Different editing images may have different mask areas, so each of them needs to be denoised from different steps.  For example, by leveraging the COCO pretrained model on 100 training steps, the "t-shirt" example needs to mask some region and denoise from 40 steps, while "tie" example needs to denoise from 10 steps. So we do not plan to release the example-specific config.

**Comment by Yikai-Wang** [comment]
I see. Can you please tell me if I correctly understand your code in the following?
If I want to run a editing task on a specific image, I should provide a mask (the same shape to the image) and the original image as part of *batch* for image_synthesis.modeling.models.dalle.DALLE.generate_content.
Besides, which value are you selecting as the mask indicator for masking the corresponding pixel, 1 or 0?
Thanks a lot!

**Comment by cientgu** [comment]
You should input a mask to generate_content function, this mask is on discrete token space, and the value is [MASK] token.
You may refer to these two files (https://drive.google.com/drive/folders/1oa4imrvkKpQv-BfgRCMwk7wROEE9qR9Q?usp=sharing), however, due to the time limit, I don't have time to clean the code and these could be ONLY used for help understanding the algorithm. We don't promise this code can be directly used in this codebase.

**Comment by Yikai-Wang** [comment]
Can you provide the corresponding file *image_synthesis.modeling.transformers.diffusion_transformer.DiffusionTransformer* ?
The *mask_token* is deleted from the released version. Thanks!

**Comment by cientgu** [comment]
I update a new file in the share folder. Hope it may help for your understanding.

**Comment by Yikai-Wang** [comment]
Thank you very much for your prompt reply. I can now understand your work very well. Thanks again for your excellent work!
